src/main/java/cn/nukkit/level/biome/impl/jungle/JungleBiome.java

```diff
@@ -20,6 +20,11 @@ public JungleBiome() {
         bigTrees.setBaseAmount(6);
         this.addPopulator(bigTrees);
 
+        PopulatorMelon melon = new PopulatorMelon();
+        melon.setBaseAmount(-65);
```

Comment:
why negative value?

Reply:
it'll only generate melons if the base value + random value is positive. this way only a few chunks will have melons in them

Reply:
Oh okey